src/server.js

```diff
@@ -1,10 +1,12 @@
 import express from "express";
 import { engine } from "express-handlebars";
 import { marked } from "marked";
+import bodyParser from "body-parser";
 
-import * as api from './api.js';
+import * as api from "./api.js";
 
 const app = express();
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
This is correctly matched against the format that the client is sending, but I would normally expect them to be JSON rather than URL-encoded form data.